cluster/charts/rook-ceph/templates/resources.yaml

```diff
@@ -1,86 +1,53 @@
 ---
-apiVersion: apiextensions.k8s.io/v1beta1
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
In the helm chart, I am hoping we can install the v1beta1 crds if it's k8s 1.15 or older, then the v1 crds if it's newer. So helm users won't even notice the difference if they're on an older version.

Reply:
Created a new file as mentioned.  How I can test helm with latest changes?